config/locales/views/search/en.yml

```diff
@@ -27,6 +27,7 @@ en:
       filter_by_pub: "Publisher"
```

Comment:
We should remove this if we're going with `filter_by_publisher`.

Reply:
Oh thank you - good spot!